perf/flame.jl

```diff
@@ -18,7 +18,7 @@ run_name_list =
 run_name = run_name_list[parse(Int, ARGS[2])]
 allocs_limit = Dict()
 allocs_limit["perf_default_modular_unthreaded"] = 8638304
-allocs_limit["perf_coarse_single_modular"] = 12671552
+allocs_limit["perf_coarse_single_modular"] = 18280800
```

Comment:
Seems to be a large increase - ~45%? Perhaps @nefrathenrici has an opinion on this?

Reply:
The allocations increase shouldn't be coming from Atmos, I think @charleskawczynski has been able to decrease allocations in recent months. 
I assume monitoring heap allocations within the coupler must be more complicated than in atmos, so I don't know if this is an immediate issue.